### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by Chaosmeister**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 1.3.1 Beta [Unreleased]
```
Comment (Chaosmeister) [inline CHANGELOG.md:8]:
```suggestion
## 1.3.1  
```

File: Plugin.php [hunk 2]
```diff
@@ -85,7 +89,7 @@ public function getPluginAuthor()
 
     public function getPluginVersion()
     {
-        return '1.3.0';
+        return '1.3.1 Beta';
```
Comment (Chaosmeister) [inline Plugin.php:92]:
```suggestion
        return '1.3.1';

```

File: Locale/de_DE/translations.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'Change position' => 'Position verschieben',
+    'Select row for deletion' => 'Zeile zum Löschen auswählen',
+    'Delete selected rows' => 'Ausgewählte Zeilen löschen',
+    'Edit row' => 'Zeile bearbeiten',
+    'Add row' => 'Zeile anfügen',
+    'Toggle state' => 'Erledigungsstatus umschalten',
+    'Close' => 'Schließen',
```
Comment (Chaosmeister) [inline Locale/de_DE/translations.php:10]:
```suggestion
    'Close row' => 'Zeile schließen',

```

File: Locale/de_DE_du/translations.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'Change position' => 'Position verschieben',
+    'Select row for deletion' => 'Zeile zum Löschen auswählen',
+    'Delete selected rows' => 'Ausgewählte Zeilen löschen',
+    'Edit row' => 'Zeile bearbeiten',
+    'Add row' => 'Zeile anfügen',
+    'Toggle state' => 'Erledigungsstatus umschalten',
+    'Close' => 'Schließen',
```
Comment (Chaosmeister) [inline Locale/de_DE_du/translations.php:10]:
```suggestion
    'Close row' => 'Zeile schließen',

```

File: Locale/es_ES/translations.php [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'Change position' => 'Mover la posición',
+    'Select row for deletion' => 'Seleccionar línea para borrar',
+    'Delete selected rows' => 'Borrar líneas seleccionadas',
+    'Edit row' => 'Editar fila',
+    'Add row' => 'Añadir línea',
+    'Toggle state' => 'Estado de finalización de conmutación',
+    'Close' => 'Cerrar',
```
Comment (Chaosmeister) [inline Locale/es_ES/translations.php:10]:
```suggestion
    'Close row' => 'Cerrar',
```

File: Locale/fr_FR/translations.php [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'Change position' => 'Modifier la position',
+    'Select row for deletion' => 'Sélection de la ligne à supprimer',
+    'Delete selected rows' => 'Effacer les lignes sélectionnées',
+    'Edit row' => 'Editer la ligne',
+    'Add row' => 'Ajouter une ligne',
+    'Toggle state' => 'Commuter l\'état d\'achèvement',
+    'Close' => 'Fermer',
```
Comment (Chaosmeister) [inline Locale/fr_FR/translations.php:10]:
```suggestion
    'Close row' => 'Fermer',
```

**Review by Chaosmeister — COMMENTED** [review]
Please apply the suggested changes, then we can merge. Thanks for your effort!

**Comment by RainerBielefeld** [comment]
Done! I hope I did not forget anything.

If you still see inconsistencies please correct here, I addee you as collaborator 

CU

Rainer

**Pull request merged**